spec/providers/social/social.double.integration.src.js

```diff
@@ -37,117 +48,82 @@ module.exports = function(provider_url, setup) {
     });
   }
 
+  function errHandler(err) {
```

Comment:
I know that one of your goals was to get rid of test utils, but this function (and maybe others like dead()) looks like it may be useful broadly enough to be worth putting somewhere once and loading it in. Not a reason not to merge this request, but possibly a reason for a "TODO" if we think it's worth doing later (as we finish standardizing tests in this new manner).